source/clear-linux/tutorials/dlrs/dlrs.rst

```diff
@@ -196,6 +196,40 @@ We used `flannel`_ as the network provider for these tests. If you
 prefer a different network layer, refer to the Kubernetes
 `networking documentation`_ for setup.
 
+Kubectl
+=======
+
+You can use kubectl to run commands against your Kubernetes cluster.  Refer to the `kubectl overview`_ for details on syntax and operations. Once you have a working cluster on Kubernetes, you can use the following YAML script to start a pod with a simple shell script, and keep the pod open.
+
+#. Copy this example.yaml yaml script to your system:
+
+   .. code-block:: console
+
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: example-pod
+        labels:
+          app: ex-pod
+      spec:
+        containers:
+        - name: ex-pod-container
+          image: clearlinux/stacks-dlrs-mkl:latest
+          command: ['/bin/bash', '-c', '--']
+          args: [ "while true; do sleep 30; done" ]
+
+#. Execute the script with kubectl:
+
+   .. code-block:: bash
+
+      kubectl apply –f <path-to-yaml-file>/example.yaml
+
+
+This will open a single pod. More robust solutions would create a deployment or inject a python script or larger shell script into the container.
```

Comment:
Change to: "This script opens a single pod."  2nd sentence is OK as-is.

Reply:
Changed!